powerline/bindings/lemonbar/powerline-lemonbar.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+# vim:fileencoding=utf-8:noet
+from __future__ import (unicode_literals, division, absolute_import, print_function)
+import sys
+import time
+import re
+import subprocess
+
+from threading import Lock, Timer
+from argparse import ArgumentParser, REMAINDER
+
+from powerline.lemonbar import LemonbarPowerline
+from powerline.lib.monotonic import monotonic
+from powerline.lib.dict import mergedicts
+from powerline.lib.shell import run_cmd
+
+
+if __name__ == '__main__':
+	parser = ArgumentParser(
+		description='Powerline BAR bindings.',
+		usage='%(prog)s [-h] [--i3] [--height HEIGHT] [--interval INTERVAL] -- args')
+	parser.add_argument(
+		'--i3', action='store_true',
+		help='Subscribe for i3 events.'
+	)
+	parser.add_argument(
+		'--height', default='',
+		help='Bar height'
+	)
+	parser.add_argument(
+		'--interval', '-i',
+		type=float, default=0.5,
+		help='Refresh interval'
+	)
+	parser.add_argument(
+		'args', nargs=REMAINDER,
+		help='Extra arguments for lemonbar'
+	)
+	args = parser.parse_args()
+
+	powerline = LemonbarPowerline()
+	powerline.update_renderer()
+	bars = []
+	active_screens = [match.groupdict() for match in re.finditer(
+		'^(?P<name>\w+) connected (?P<width>\d+)x(?P<height>\d+)\+(?P<x>\d+)\+(?P<y>\d+)',
+		run_cmd(powerline.pl, ['xrandr', '-q']),
+		re.MULTILINE
+	)]
+
+	for screen in active_screens:
+		command = ['bar-aint-recursive', '-g', '{}x{}+{}'.format(screen['width'], args.height, screen['x'])] + args.args[1:]
```

Comment:
Also you must not use `{}`. This is not supported on Python-2.6, use explicit argument numbers.